### DISCUSSION TITLE
Fermata control

### DISCUSSION THREAD

**Discussion by Salafi83** [AndreyPavlenko/Fermata#241] [General]
Does fermata control allow YouTube videos to be played in the background?

**Comment by AndreyPavlenko** [comment]
Fermata Control can't play YouTube at all.

**Comment by Salafi83** [comment]
Is there way to get fermata to play in the background?

**Comment by AndreyPavlenko** [comment]
To be added in the next version of Fermata Auto.

**Comment by Salafi83** [comment]
When is it going to be released?

**Comment by AndreyPavlenko** [comment]
Released.

**Comment by Salafi83** [comment]
Awesome. Thanks!
